### ISSUE TITLE
Schema gets weird \" escapes from Yaml parsing

### ISSUE THREAD

**Issue by lawik** [wolfadex/elm-open-api-cli#53]
I have no idea why this happens on my schema and not on the github-spec.json. But it is consistent:

```
{
  "components": {
    "responses": {},
    "schemas": {}
  },
  "info": {
    "title": "Sample",
    "version": "0.1"
  },
  "openapi": "3.0.0",
  "paths": {},
  "security": [],
  "servers": [],
  "tags": []
}
```

Removing the option for yaml and forcing a simple Json.decode works great.

Running the code as is gives this:

```
-- CUSTOM ERROR --------------- 
Problem with the value at json.openapi:

    "\"3.0.0\""

Invalid version format
```

**Comment by lawik** [comment]
Okay, I've confirmed that if the yaml parser fails (as it does on the github example), then everything goes well. If the yaml parser succeeds it escapes my strings.

**Comment by lawik** [comment]
The github example fails on:

```
Parsing "Line 24, column 83: Encountered an unexpected character\nLine 24, column 83: Problem: I was parsing an inline record, when I ran into an invalid property. It is missing the \":\"!"
```

Mine clears as yaml just fine unfortunately and gets escaped.

**Comment by lawik** [comment]
Or rather, the quotes are kept around for some reason. The quotes are there inside of Yaml.Decode.String which is used in yamlToJsonDecoder.

**Comment by wolfadex** [comment]
This definitely is a weird bug. By chance would you be able to see if https://package.elm-lang.org/packages/MaybeJustJames/yaml/latest/ has the same parsing issue? I'm looking to switch to it as part of issue #38. Being able to resolve 2 bugs with 1 fix would be nice.

**Comment by lawik** [comment]
Oh yeah, it reproduces up to lates main on the yaml-package. But I think my PR is a reasonably harmless fix.

**Comment by lawik** [comment]
Release version `0.1.1` does not seem to have built the actual code into it.

Looking through the build in dist it does not contain the string ".json" which it does if I force a build locally. @wolfadex probably something to look at :)

**Comment by wolfadex** [comment]
> Release version `0.1.1` does not seem to have built the actual code into it.
> 
> Looking through the build in dist it does not contain the string ".json" which it does if I force a build locally. @wolfadex probably something to look at :)

Sorry about that. I forgot to build before publishing (I should set that up so I don't forget again in the future).  I've published `0.1.2` which should contain the fix. I'll leave this open till you let me know it's working for you.

**Comment by lawik** [comment]
Sweet, thanks. Confirmed working.
